app/src/main/java/com/example/plmarket/media/data/PlayListConvertor.kt

```diff
@@ -0,0 +1,44 @@
+package com.example.plmarket.media.data
+
+import com.example.plmarket.media.data.db.entity.PlayListEntity
+import com.example.plmarket.media.data.db.entity.TrackPlayListEntity
+import com.example.plmarket.media.domain.module.PlayList
+import com.example.plmarket.player.domain.models.Track
+
+class PlayListConvertor {
+
+    fun map(playList: PlayList): PlayListEntity {
+        return PlayListEntity(
+            playList.playListId,
+            playList.name,
+            playList.description.toString(),
+            playList.uri,
+            playList.currentTracks
+        )
+    }
```

Comment:
Подобные мапперы можно писать более кратко и лаконично
```suggestion
    fun map(playList: PlayList): PlayListEntity = PlayListEntity(
            playList.playListId,
            playList.name,
            playList.description.toString(),
            playList.uri,
            playList.currentTracks
        )
```